2023-01-19 09:34:49.900 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration dwains_dashboard which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:34:49.911 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration xiaomi_cloud_map_extractor which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:34:49.919 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration ui_lovelace_minimalist which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:34:49.927 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:34:49.935 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration rte_ecowatt which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:35:37.553 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform command_line is taking over 10 seconds.
2023-01-19 09:35:37.620 WARNING (MainThread) [homeassistant.setup] Setup of input_datetime is taking over 10 seconds.
2023-01-19 09:35:37.626 WARNING (MainThread) [homeassistant.setup] Setup of input_select is taking over 10 seconds.
2023-01-19 09:35:37.632 WARNING (MainThread) [homeassistant.setup] Setup of input_text is taking over 10 seconds.
2023-01-19 09:35:37.638 WARNING (MainThread) [homeassistant.setup] Setup of input_number is taking over 10 seconds.
2023-01-19 09:35:37.645 WARNING (MainThread) [homeassistant.setup] Setup of input_boolean is taking over 10 seconds.
2023-01-19 09:35:37.651 WARNING (MainThread) [homeassistant.setup] Setup of input_button is taking over 10 seconds.
2023-01-19 09:35:37.657 WARNING (MainThread) [homeassistant.setup] Setup of schedule is taking over 10 seconds.
2023-01-19 09:35:37.664 WARNING (MainThread) [homeassistant.setup] Setup of zone is taking over 10 seconds.
2023-01-19 09:36:02.927 WARNING (MainThread) [homeassistant.components.binary_sensor] Setup of binary_sensor platform xiaomi_miio is taking over 10 seconds.
2023-01-19 09:36:02.934 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform xiaomi_miio is taking over 10 seconds.
2023-01-19 09:36:02.942 WARNING (MainThread) [homeassistant.components.camera] Setup of camera platform xiaomi_cloud_map_extractor is taking over 10 seconds.
2023-01-19 09:36:02.948 WARNING (MainThread) [homeassistant.components.binary_sensor] Setup of binary_sensor platform rpi_power is taking over 10 seconds.
2023-01-19 09:36:02.954 WARNING (MainThread) [homeassistant.components.vacuum] Setup of vacuum platform xiaomi_miio is taking over 10 seconds.
2023-01-19 09:36:02.984 WARNING (SyncWorker_7) [homeassistant.components.rpi_power.binary_sensor] Under-voltage was detected. Consider getting a uninterruptible power supply for your Raspberry Pi.
2023-01-19 09:36:07.917 WARNING (MainThread) [homeassistant.config_entries] Config entry 'LG webOS TV 214E' for homekit_controller integration not ready yet: Timeout while waiting for connection to device 192.168.1.113:34734; Retrying in background
2023-01-19 09:36:09.010 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2023-01-19 09:36:09.290 ERROR (MainThread) [homeassistant.components.sensor] Platform rte_ecowatt does not generate unique IDs. ID ecowatt-level-in-0-hours already exists - ignoring sensor.ecowatt_level_now
2023-01-19 09:36:19.058 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform rte_ecowatt is taking over 10 seconds.
2023-01-19 09:36:20.756 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up rte_ecowatt platform for sensor
Traceback (most recent call last):
  File "/config/custom_components/rte_ecowatt/__init__.py", line 187, in update_method
    raise UpdateFailed(
homeassistant.helpers.update_coordinator.UpdateFailed: Error communicating with RTE API: requests too frequent to RTE API

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 225, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 181, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/rte_ecowatt/__init__.py", line 206, in update_method
    raise UpdateFailed(f"Error communicating with API: {err}")
homeassistant.helpers.update_coordinator.UpdateFailed: Error communicating with API: Error communicating with RTE API: requests too frequent to RTE API

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 289, in _async_setup_platform
    await asyncio.shield(task)
  File "/config/custom_components/rte_ecowatt/sensor.py", line 98, in async_setup_entry
    await rte_coordinator.async_config_entry_first_refresh()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 197, in async_config_entry_first_refresh
    raise ex
homeassistant.exceptions.ConfigEntryNotReady: Error communicating with API: Error communicating with RTE API: requests too frequent to RTE API
2023-01-19 09:36:28.546 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2023-01-19 09:37:03.383 ERROR (MainThread) [homeassistant.components.xiaomi_miio] Timeout fetching Victor data
2023-01-19 09:37:03.469 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2023-01-19 09:37:18.302 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2023-01-19 09:39:38.357 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2023-01-19 09:40:07.981 WARNING (SyncWorker_12) [homeassistant.components.rpi_power.binary_sensor] Under-voltage was detected. Consider getting a uninterruptible power supply for your Raspberry Pi.
2023-01-19 09:41:30.729 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration dwains_dashboard which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:41:30.854 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration xiaomi_cloud_map_extractor which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:41:30.909 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hyperhdr which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:41:30.997 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration ui_lovelace_minimalist which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:41:31.084 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:41:31.169 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration rte_ecowatt which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-19 09:41:55.302 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /homeassistant/restart request Server disconnected
2023-01-19 09:41:55.426 WARNING (Thread-7 (_do_shutdown)) [homeassistant.util.executor] Thread[SyncWorker_9] is still running at shutdown: File "/usr/local/lib/python3.10/threading.py", line 973, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.10/concurrent/futures/thread.py", line 83, in _worker
    work_item.run()
  File "/usr/local/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/components/xiaomi_miio/__init__.py", line 245, in update
    device.dnd_status(),
  File "/usr/local/lib/python3.10/site-packages/miio/click_common.py", line 184, in _wrap
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/miio/integrations/vacuum/roborock/vacuum.py", line 612, in dnd_status
    return DNDStatus(self.send("get_dnd_timer")[0])
  File "/usr/local/lib/python3.10/site-packages/miio/device.py", line 107, in send
    return self._protocol.send(
  File "/usr/local/lib/python3.10/site-packages/miio/miioprotocol.py", line 193, in send
    data, addr = s.recvfrom(4096)
